engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -791,6 +176,11 @@ public float getExposure() {
         return currentExposure;
     }
 
+    // TODO: Remove this method, temporarly here for DownSampleSceneAndUpdateExposure
```

Comment:
I am open to that. I'd like to refresh my knowledge about those classes and their interactions with shaders. I might also ask @immortius opinion on this as I seem to remember he has been working on the shader/materials/parameter classes.